mmv1/products/dataplex/AspectType.yaml

```diff
@@ -0,0 +1,144 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'AspectType'
+base_url: 'projects/{{project}}/locations/{{location}}/aspectTypes/{{aspect_type_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/aspectTypes/{{aspect_type_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/aspectTypes?aspectTypeId={{aspect_type_id}}'
+update_url: 'projects/{{project}}/locations/{{location}}/aspectTypes/{{aspect_type_id}}'
```

Comment:
Nit: Same for update_url and delete_url, I think they default to self_link, and these lines can be removed